docs/src/pages/options/installing-icon-libraries.md

```diff
@@ -112,12 +117,12 @@ chip: {
 _Then_, override it in your `/src/boot/fontawesome-pro.js`
 
 ```js
-import Vue from 'vue'
-
 import '@fortawesome/fontawesome-pro/css/fontawesome.min.css'
 import '@fortawesome/fontawesome-pro/css/solid.min.css'
 import '@fortawesome/fontawesome-pro/css/light.min.css'
 
 // example
-Vue.prototype.$q.iconSet.chip.remove = 'fas fa-times-circle'
+export default ({ app }) => {
```

Comment:
Should actually be Vue.prototype.$q.... This is Qv1 on Vue 2.

Reply:
Thanks for the fix.